pyoffers/models/application.py

```diff
@@ -0,0 +1,24 @@
+from pyoffers.utils import Filter
```

Comment:
``from ..utils import Filter``

Reply:
this is no longer actual since this model has moved to core